CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ HumHub Changelog
 - Enh #7167: Disable DEBUG mode automatically after successful humhub installation. Add `.env` support
 - Enh #7202: Increased minimum PHP version to 8.1
 - Enh #7211: Remove obsolete property `AbstractRichText::$record`
+- Enh #221: By default, enabled "Mobile" notification types for those enabled for "Web"
```

Comment:
#7213

Reply:
Done in commit https://github.com/humhub/humhub/pull/7213/commits/72bbbc1ecc8161fa9d4f566ee1925d908578f599